### PR TITLE
[improve][doc] add explanations of how to expose bundle metrics

### PR DESCRIPTION
**Pull request by Anonymitaet**
This PR fixes https://github.com/apache/pulsar/issues/20881

@Demogorgon314 thanks for your technical guidance!

- [x] `doc`

### CODE REVIEW

**Comment by Anonymitaet** [comment]
Preview looks good:
image

**Comment by Anonymitaet** [comment]
Ping @Demogorgon314 @heesung-sn @@D-2-Ed  to review, thanks!

File: docs/concepts-broker-load-balancing-concepts.md [hunk 1]
```diff
@@ -544,9 +544,9 @@ TransferShedder strategy unloads bundles from the **highest** load brokers to th
 
     - No broker's load < avgLoad * min(0.5, loadBalancerBrokerLoadTargetStd / 2)
 
-    - There is no significant overloaded brokers
+- There is no significant overloaded brokers
 
-      - No broker’s load > loadBalancerBrokerOverloadedThresholdPercentage && load > avgLoad + loadBalancerBrokerLoadTargetStd
+    - No broker’s load > loadBalancerBrokerOverloadedThresholdPercentage && load > avgLoad + loadBalancerBrokerLoadTargetStd
```
Comment (Anonymitaet) [inline docs/concepts-broker-load-balancing-concepts.md:549]: @heesung-sn fix the layout issue here

**Review by Demogorgon314 — APPROVED** [review]

**Comment by Anonymitaet** [comment]
Updated preview:
image

**Pull request merged**